asyncudp/__init__.py

```diff
@@ -21,10 +23,18 @@ def datagram_received(self, data, addr):
         self._packets.put_nowait((data, addr))
 
     def error_received(self, exc):
-        pass
+        self._error = exc
```

Comment:
What about handling this error in the same way as connection lost? That is, just put None on the packets queue. Could possibly save the exception, as done above, in combination with putting None on the packets queue.

```python
def error_received(self, exc):
    self._packets.put_nowait(None)
```